active_learning_multiannotator/active_learning.ipynb

```diff
@@ -0,0 +1,419 @@
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "File ‘data.tar.gz’ already there; not retrieving.\r\n",
+      "\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/ActiveLearning/WallRobot/data.tar.gz'\n",
+    "!tar -xf data.tar.gz data/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will load the following datafiles:\n",
+    "\n",
+    "- `multiannotator_labels` is a DataFrame that contains labels from multiple annotators for each datapoint\n",
+    "- `X_labeled` are the features for the examples that have been labeled by at least one annotator\n",
+    "- `X_unlabeled` are the features for the examples that have not been labeled by any annotator\n",
+    "\n",
+    "Additionally, we load these extra files for simulation and demontration purposed and would usually not be available:\n",
+    "\n",
+    "- `extra_labels_labeled` and `extra_labels_unlabeled` are synthetically created extra labels that will be sampled from when we get more labels later in this notebook\n",
+    "- `X_test` and `true_labels_test` are used to measure the model accuracy "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multiannotator_labels = pd.DataFrame(np.load(\"data/multiannotator_labels_labeled.npy\"))\n",
+    "\n",
+    "X_labeled = np.load(\"data/X_labeled.npy\")\n",
+    "X_unlabeled = np.load(\"data/X_unlabeled.npy\")\n",
+    "\n",
+    "extra_labels_labeled = np.load(\"data/extra_labels_labeled.npy\")\n",
+    "extra_labels_unlabeled = np.load(\"data/extra_labels_unlabeled.npy\")\n",
+    "\n",
+    "X_test = np.load(\"data/X_test.npy\")\n",
+    "true_labels_test = np.load(\"data/true_labels_test.npy\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train model and obtain predicted probabilites\n",
+    "\n",
+    "First, we train our model on a set of consensus labels obtained using majority vote to get the out-of-sample predicted probabilities for both the labeled and unlabeled data. \n",
+    "\n",
+    "The function will return two sets of predicted probabilites, `pred_probs_labeled` are the probabilites for examples that have existing annotator labels (they correspond directly with the features `X_labeled`), whereas `active_learning_score_unlabeled` are the probabilites for examples that do not have any annotator labels (they correspond directly with the features `X_unlabeled`). These predicted probabilities will later be used to compute the active learning score.\n",
+    "\n",
+    "Note that our `train_model` function here also computes the model accuracy however if you have no test data, you can set `X_test=None` and `labels_test=None` and the function will still compute the predicted probabilites."
```

Comment:
`train_model` should perhaps be called `fit_predict` like sklearn terminology? (don't recall exact sklearn terminology so you could look into the most similar functions they have)